src/main/cli/settings.c

```diff
@@ -1544,6 +1544,7 @@ const clivalue_t valueTable[] = {
 #endif
     { "pwr_on_arm_grace",           VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 30 }, PG_SYSTEM_CONFIG, offsetof(systemConfig_t, powerOnArmingGraceTime) },
     { "enable_stick_arming",        VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_SYSTEM_CONFIG, offsetof(systemConfig_t, enableStickArming) },
+    { "stats_saving_max_gyro_rate", VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 255 }, PG_SYSTEM_CONFIG, offsetof(systemConfig_t, statsSavingMaxGyroRate) },
```

Comment:
```suggestion
    { "stats_move_threshold", VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 255 }, PG_SYSTEM_CONFIG, offsetof(systemConfig_t, statsSavingMovementThreshold) },
```
I wonder if we can simplify this CLI command because it's so long. In my opinion "gyro" is too specific in case we include other sensors in the future. So maybe `stats_move_threshold`?

Reply:
i made it `stats_save_move_limit`.... so it was clear its about saving :)